### DISCUSSION TITLE
Latest 1.1.0 flagged by windows defender as virus

### DISCUSSION THREAD

**Discussion by carlomazza** [bdd/runitor#77] [Q&A]
Previous 1.0.0 generated warning, but this is quickly deleted because of Trojan:Win32/Wacatac.B!ml

**Comment by bdd** [comment]
VirusTotal doesn't include Windows Defender, but all other AV software seems oblivious to it.
https://www.virustotal.com/gui/url/28e01e5e54fe75815ea8ed41820cf7b4e16dd6d6b360719c2b310c771612f165?nocache=1

**Comment by bdd** [comment]
@carlomazza
- fair to assume you downloaded the release from github?
- did you check its SHA-256 and compare it to the one in SHA256 file?
- can you share a screenshot of Windows Defender's message?
- are there logs of Windows Defender actions, if so, can you also share them?

**Comment by carlomazza** [comment]
Yes, I downloaded it from github. I did not check the sha256, will do. In the meantime I attach two screenshots: one for the 1.1.0 that was deleted immediately, and one for the 1.0.0 which was allowed yesterday, but put in quarantine today as soon as I turned on the computer.
Windows defender redirects for infos to https://www.microsoft.com/en-us/wdsi/threats/malware-encyclopedia-description?name=Trojan%3aWin32%2fWacatac.B!ml&threatid=2147735505
![2](https://user-images.githubusercontent.com/139716/203617100-c3b4741b-650c-47ef-94fa-d83788a7f7c0.PNG)
![1](https://user-images.githubusercontent.com/139716/203617104-b09f5a6e-3c05-46ad-8a57-1c0cf80f59ab.PNG)

**Reply by carlomazza** [reply]
... and 1.1.1-rc1 did not generate any particular message, if not for the fact that is downloaded not often (I attach screenshot). So all is well, at least for now...
![3](https://user-images.githubusercontent.com/139716/203617929-4b5f662f-048a-4c59-b524-75c766c49d53.PNG)

**Reply by bdd** [reply]
I really have no idea why this is happening.
Wouldn't know how to fix it either.
Let's hope this was a false positive and somehow the Defender definition update fixed it.
